android/res/values/strings-tts.xml

```diff
@@ -13,55 +13,65 @@
     Otherwise we consider that this is zh-Hans.
 
     TODO: Move language list to core.
+
+    languages should be added alphabetically in tts_language_names
   -->
   <string-array name="tts_languages_supported"
                 translatable="false">
     <item>en</item>
-    <item>ru</item>
+    <item>id</item>
     <item>ca</item>
-    <item>cs</item>
     <item>da</item>
     <item>de</item>
     <item>es</item>
+    <item>eu</item>
     <item>fr</item>
-    <item>id</item>
+    <item>hr</item>
     <item>it</item>
+    <item>sw</item>
     <item>hu</item>
     <item>nl</item>
+    <item>nb</item>
     <item>pl</item>
     <item>pt</item>
     <item>ro</item>
-    <item>sk</item>
+    <item>sv</item>
```

Comment:
@endim8 sv is Swedish, but below it was Slovenčina

Reply:
thanks, fixed in https://github.com/organicmaps/organicmaps/pull/4308 :)